ng-rollbar.js

```diff
@@ -2,10 +2,17 @@
   angular.module('tandibar/ng-rollbar', []);
 
   angular.module('tandibar/ng-rollbar').config(['$provide', function($provide) {
-    $provide.decorator('$exceptionHandler', ['$delegate', '$window', function($delegate, $window) {
+    $provide.decorator('$exceptionHandler', ['$delegate', '$injector', '$window', function($delegate, $injector, $window) {
       return function (exception, cause) {
         if($window.Rollbar) {
-          $window.Rollbar.error(exception, {cause: cause});
+          $window.Rollbar.error(exception, {cause: cause}, function(err, data) {
+            var $rootScope = $injector.get('$rootScope');
```

Comment:
In the decorator, `$rootScope` must be fetched from the `$injector` since injecting `$rootScope` into the decorator itself causes a circular dependency error.